bottlecap/Cargo.toml

```diff
@@ -15,7 +15,7 @@ datadog-trace-utils = { git = "https://github.com/DataDog/libdatadog", rev = "eb
 datadog-trace-mini-agent = { git = "https://github.com/DataDog/libdatadog", rev = "eb82b62002a897a8bb9fe1e1cfa49170a9ea6b21" }
 datadog-trace-normalization = { git = "https://github.com/DataDog/libdatadog", rev = "eb82b62002a897a8bb9fe1e1cfa49170a9ea6b21" }
 datadog-trace-obfuscation = { git = "https://github.com/DataDog/libdatadog", rev = "eb82b62002a897a8bb9fe1e1cfa49170a9ea6b21" }
-dogstatsd = { git = "https://github.com/DataDog/libdatadog", rev = "eb82b62002a897a8bb9fe1e1cfa49170a9ea6b21" }
+dogstatsd = { git = "https://github.com/DataDog/libdatadog", rev = "5bba5f1415fa07a5a2fc04ec6a3a533909dda1e5" }
```

Comment:
This is fine but let's bump it to the commit once you merge to main in libdatadog